docs/slack.rst

```diff
@@ -265,25 +267,29 @@ In addition to the guidance that applies to all participants, we expect self-pro
   The Write the Docs Slack is informal and conversational; high-pressure sales aren't welcome.
   If people aren't interested in what you're offering, don't take it as a license to become louder and more aggressive.
   If you're asked to stop, then stop.
+- **Treat the community like friends, not customers.**
+  Some posts can come off as infomercials, and those are harder for the community to engage with and for the moderation team to distinguish from spam.
```

Comment:
(optional, style)

"can come off as" struck me a potentially fraught construction in this context, particularly for folks less fluent in English.

(Also inspired by my recent feeling of being in some sort of Greek myth: a man commits hubris by using too many phrasal verbs, so the gods condemn him to learn Dutch.)

```suggestion
  Posts that resemble infomercials are harder for the community to engage with and for the moderation team to distinguish from spam.
```